Cargo.toml

```diff
@@ -25,26 +25,23 @@ default = []
 api_client = ["racal/reqwest", "reqwest", "governor", "percent-encoding", "base64", "async-trait"]
 
 [dependencies]
-serde = { version = "1.0", features = ["derive"] }
-serde_with = { version = "2.1", features = ["time_0_3"] }
-time = { version = "0.3", default-features = false, features = [
-	"macros",
-	"serde-well-known",
-] }
+serde = { version = "1", features = ["derive"] }
+serde_with = { version = "3", features = ["time_0_3"] }
```

Comment:
I'm not sure what all has changed since v2 of serde_with, but it doesn't seem to break anything in my limited testing. I did check to see if it was the reason Either no longer serializes, but no, none of the dependency upgrades caused that.